lib/utils/ln_payment_validator_utils.dart

```diff
@@ -0,0 +1,9 @@
+class LnPaymentValidatorUtils {
+  String formatLnPaymentValidatorUrl({
+    required String invoice,
+    required String preimage,
+    String lnPaymentValidator = 'https://validate-payment.netlify.app/',
```

Comment:
Should it be https://validate-payment.com/?
```suggestion
    String lnPaymentValidator = 'https://validate-payment.com/',
```